### ISSUE TITLE
NULL was treated differently from other DB when being orderred

### ISSUE THREAD

**Issue by wey-gu** [vesoft-inc/nebula#4821]
**Please check the FAQ documentation before raising an issue**

As title, we treated NULL differently from other DB, which is counterintuitive to user.

```
(root@nebula) [basketballplayer]> match (n:player) return n.player.name AS name order by name DESC limit 2
+------------+
| name       |
+------------+
| __NULL__   |
| "Yao Ming" |
+------------+
Got 2 rows (time spent 2.626ms/3.188925ms)
```

<img width="1015" alt="Screen Shot 2022-11-03 at 10 02 48" src="https://user-images.githubusercontent.com/1651790/199634948-afea983f-6c87-4fa1-ae49-8f8cd80900bf.png">

**Describe the bug (__required__)**

**Your Environments (__required__)**

* OS: `uname -a`
* Compiler: `g++ --version` or `clang++ --version`
* CPU: `lscpu`
* Commit id (e.g. `a3ffc7d8`)

**Expected behavior**

NULL should be at the end when DESC.

**Additional context**

ref: https://github.com/vesoft-inc/nebula/issues/4810

**Comment by HarrisChu** [comment]
![image](https://user-images.githubusercontent.com/1726587/199635729-aaf3f9fd-9976-40ac-9762-b63ebb1a9a45.png)

**Comment by HarrisChu** [comment]
BTW, PostgreSQL could special the behavior ( null first or null last)
```
SELECT select_list
    FROM table_expression
    ORDER BY sort_expression1 [ASC | DESC] [NULLS { FIRST | LAST }]
             [, sort_expression2 [ASC | DESC] [NULLS { FIRST | LAST }] ...]
```

**Comment by wey-gu** [comment]
> BTW, PostgreSQL could special the behavior ( null first or null last)
> 
> ```
> SELECT select_list
>     FROM table_expression
>     ORDER BY sort_expression1 [ASC | DESC] [NULLS { FIRST | LAST }]
>              [, sort_expression2 [ASC | DESC] [NULLS { FIRST | LAST }] ...]
> ```

Thanks! Exposing options is way more elegant!

**Comment by wey-gu** [comment]
> ![image](https://user-images.githubusercontent.com/1726587/199635729-aaf3f9fd-9976-40ac-9762-b63ebb1a9a45.png)

OK, now we know we are not wrong, just one flavor of all approaches :)
Thanks!

**Comment by wey-gu** [comment]
> ![image](https://user-images.githubusercontent.com/1726587/199635729-aaf3f9fd-9976-40ac-9762-b63ebb1a9a45.png)

@son2408 it seems current behavior is one of the approaches that PostgreSQL/Oracle also behaves by default

**Comment by son2408** [comment]
> > ![image](https://user-images.githubusercontent.com/1726587/199635729-aaf3f9fd-9976-40ac-9762-b63ebb1a9a45.png)
> 
> @son2408 it seems current behavior is one of the approaches that PostgreSQL/Oracle also behaves by default

Yes, we are not wrong, it should be extra option  NULL fisrt or last :)
